### PR TITLE
Email Dashboard!

### PR DESCRIPTION
**Pull request by aidanouckama**
Unfortunately, this isn't a feature for the average Pipeliner, this is for us developers to spam you guys with emails.

**Dashboard Demon**
`client/src/pages/(admin)/AdminDashboard.js` - Removed the company section to add a hub for different pages. 
`client/src/pages/(admin)/pages/*` - Sectioned off pages into its own directory.
[ NEW ] `client/src/pages/(admin)/pages/EmailDashboard.js` - New Email Dashboard!
`server/controllers/emailController.js` - Added new routes to handle sending emails to certain emails / groups.
`server/routes/emails.js` - Separated router into `read` and `write` routers, write having admin access and read having user access.
`server/index.js` - Added admin verification to email routes (except Newsletter of course).

**Bug Squishin**
`client/src/pages/Newsletter.js` - Added a slight delay to the Newsletter page to prevent spamming.

image

LONG LIVE PIPELINES! 🚀

### CODE REVIEW

File: client/src/pages/Newsletter.js [hunk 1]
```diff
@@ -19,12 +19,15 @@ function Newsletter() {
     const handleSubmit = async (e) => {
         e.preventDefault()
 
+        // Disable the submit button to prevent spamming
+        e.target.disabled = true
```
Comment (Lewin-B) [inline client/src/pages/Newsletter.js:23]: nice

**Review by Lewin-B — CHANGES_REQUESTED** [review]
Mostly good it's just that the sidebar on the dashboard is cooked for me

Screenshot 2024-04-09 at 11 50 04 AM

**Comment by aidanouckama** [comment]
> Mostly good it's just that the sidebar on the dashboard is cooked for me
> 
> Screenshot 2024-04-09 at 11 50 04 AM

Yeah I saw that happen when I'd have the console open, I'll fix it real quick.

**Review by Lewin-B — APPROVED** [review]
lgtm

**Pull request merged**